.github/workflows/elixir.yml

```diff
@@ -2,9 +2,9 @@ name: CI
 
 on:
   push:
-  pull_request:
     branches:
       - main
+  pull_request:
```

Comment:
Remove the duplicate check from 26 to 13 by limiting it to pull request branch branch only instead of adding `main` branch as well.